backend/src/index.ts

```diff
@@ -205,8 +205,8 @@ const main = async () => {
   app.use("/api/v1/organizations", eeOrganizationsRouter);
   app.use("/api/v1/sso", eeSSORouter);
   app.use("/api/v1/cloud-products", eeCloudProductsRouter);
-  app.use("/api/v3/api-key", v3apiKeyDataRouter); // new
-  app.use("/api/v3/service-token", v3ServiceTokenDataRouter); // new
+  app.use("/api/v3/api-key", v3apiKeyDataRouter);
+  app.use("/api/v3/machines", v3MachineIdentityRouter);
```

Comment:
this might fit well under auth route? Maybe something like this:

/api/v1/auth/machine-identity/gcp

/api/v1/auth/machine-identity/kubernetes

/api/v1/auth/machine-identity/aws

/api/v1/auth/machine-identity/machinerole